models/autotetraploid.slim

```diff
@@ -0,0 +1,130 @@
+// Peter Ralph, 27 June 2019: autotetraploidy
+
+/*
+	Tetraploidy:
+	
+	This is a simple model of an autotetraploid.  To do this, we have to 
+	(a) give each individual *four* genomes
+	(b) let crossing-over happen between any of the possible pairings
+	(c) compute fitness with all four genomes
```

Comment:
Ah, I thought you hadn't dealt with fitness; I thought it was just a neutral model.  Wow, nice.